sherpa/astro/io/__init__.py

```diff
@@ -228,7 +224,9 @@ def read_table(arg, ncols=2, colkeys=None, dstype=Data1D):
     >>> d = read_table('tbl.fits', colkeys=['WLEN', 'FLUX', 'FLUXERR'])
 
     """
-    colnames, cols, name, hdr = backend.get_table_data(arg, ncols, colkeys)
+    args = backend.get_table_data(arg, ncols, colkeys)
+    cols = args[1]
+    name = args[2]
```

Comment:
I prefer the old version for readability, but that's personal taste, so I won't argue either way.

Reply:
Well, pylint didn't like the unused colnames and hdr variable settings. I could have gone with the `_, cols, name, _ = ...` trick, but then it feels like I'm writing Haskell. We're also a ways off getting destructuring in the language (Python 3.10).